common/lib/capa/capa/xqueue_interface.py

```diff
@@ -15,6 +15,8 @@
 
 # Wait time for response from Xqueue.
 XQUEUE_TIMEOUT = 35  # seconds
+CONNECT_TIMEOUT = 3.05  # seconds
```

Comment:
I think @noraiz-anwar made a comment about renaming this to `CONNECTION_TIMEOUT`. I think that makes sense!